addons/point_of_sale/static/src/js/models.js

```diff
@@ -2173,7 +2173,7 @@ exports.Orderline = Backbone.Model.extend({
             var pack_lot_line = new exports.Packlotline({}, {'json': _.extend({...packlotline}, {'order_line':this})});
             this.pack_lot_lines.add(pack_lot_line);
         }
-        this.tax_ids = json.tax_ids && json.tax_ids.length !== 0 ? json.tax_ids[0][2] : undefined;
```

Comment:
There's a reason why this has been done this way. See export_as_JSON

Reply:
Hello @trj-odoo 

yes there it's a reason, it's because tax_ids is always treated as an array of ids,
this line in particularity was expecting json.tax_ids as a list of tuples
```
[[0, 0, tax]]
```

but tax_ids was never loaded at all, so L2176 never has any effect

in case of export_as_JSON,  it's needed that way because it's only used for saving the order in backend

Reply:
export_as_JSON is also used to store in localStorage so you will have an issue when loading from it

Reply:
thanks! you right!

Done